### PR TITLE
Minor typo fixes

### PR DESCRIPTION
**Pull request by dwinters99**
Cowled vests aren't from the future anymore.
W isn't a vowel anymore.
~~De-russian'd syntax in webbing/drop-pouches description.~~ 

Hopefully I can stop hecking this up.

EDIT: **PLEASE** notify me of any other typos you know of, hopefully I can catch them _before_ and _if_ this is merged.

### CODE REVIEW

**Comment by KasparoVy** [comment]
> /me lets out an woof

LMAO, at first I thought it was intentional in a jokey sort of way. Love to see that throwback Gmod syntax as well

**Comment by dwinters99** [comment]
Forgetting to close a pull request is evil. I'm fixing it as recompense.

File: code/modules/mob/living/carbon/human/emote_vr.dm [hunk 1]
```diff
@@ -40,11 +40,11 @@
 			playsound(src, 'sound/voice/growl.ogg', 50, 1, -1, preference = /datum/client_preference/emote_noises)
 		if("woof")
 			m_type = 2
-			message = "lets out an woof."
+			message = "lets out a woof."
```
Comment (KasparoVy) [inline code/modules/mob/living/carbon/human/emote_vr.dm:43]: rip an woof, every time I read this I think "moon moon"

**Pull request merged**